rackspace/metrics/v2/ingestion/requests.go

```diff
@@ -0,0 +1,205 @@
+package ingestion
+
+import (
+	"github.com/rackspace/gophercloud"
+	"fmt"
+)
+
+// SendMetrics will ingest the metrics for the tenant associated with RackspaceMetrics.
+func SendMetrics(c *gophercloud.ServiceClient, metrics []MetricData) {
+	var res PostResult
+	reqBody := make([]map[string]interface{}, len(metrics))
+	for i := range metrics {
+		Metric := metrics[i]
+		reqBody[i] = map[string]interface{}{
+			"collectionTime": Metric.CollectionTime,
+			"ttlInSeconds":Metric.TtlInSeconds,
+			"metricValue": Metric.MetricValue,
+			"metricName": Metric.MetricName,
```

Comment:
A lot of these look like `struct`s you could define along with tags for what they serialize to in JSON. There's a lot of repetition here for fields.